src/Microsoft.DotNet.Files/ContentFiles.cs

```diff
@@ -19,45 +20,68 @@ public ContentFiles(ProjectContext context)
             _context = context;
         }
 
-        public void StructuredCopyTo(string targetDirectory)
+        public void StructuredCopyTo(string targetDirectory, IEnumerable<IncludeEntry> includeEntries)
         {
-            var sourceFiles = _context
-                .ProjectFile
-                .Files
-                .GetContentFiles();
+            if (includeEntries == null)
```

Comment:
it feels like the logics in if and else can be separated into two different methods. the client can all the different function just base on if the `includeEntries` is null.